awslogs/bin.py

```diff
@@ -189,20 +189,19 @@ def add_date_range_arguments(parser, default_start='5m'):
         options['aws_session_token'] = 'SENSITIVE'
         options['aws_profile'] = 'SENSITIVE'
         sys.stderr.write("\n")
-        sys.stderr.write("=" * 80)
         sys.stderr.write("\nYou've found a bug! Please, raise an issue attaching the following traceback\n")
         sys.stderr.write("https://github.com/jorgebastida/awslogs/issues/new\n")
-        sys.stderr.write("-" * 80)
         sys.stderr.write("\n")
+        sys.stderr.write("```")
```

Comment:
Robust (paranoid?) approach to backticks for the preformatted block:
```python
block = ...
backticks = 3 + max(
    (len(match.group()) for match in re.finditer(r"`{3,}", block)),
    default=0,
)
sys.stderr.write("`" * backticks + "\n")
sys.stderr.write(block + "\n")
sys.stderr.write("`" * backticks + "\n")
```

Reply:
@Code0x58 
Interesting.  Please consider creating a PR with that.